integration/provider.go

```diff
@@ -0,0 +1,55 @@
+package integration
+
+import (
+	"autobot_integration/pkg"
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func Provider() *schema.Provider {
+	return &schema.Provider{
+		Schema: map[string]*schema.Schema{
+
+			"apikey": {
+				Type:        schema.TypeString,
+				Sensitive:   true,
+				Required:    true,
+				DefaultFunc: schema.EnvDefaultFunc("APIKEY", nil),
```

Comment:
Put this in the documentation somewhere that you can either set the key in variable or set as environment variable.